README.md

```diff
@@ -53,27 +56,47 @@ You can install the app using one of the following methods:
 Build the app from sources in a few simple steps:
 
 1. Clone the repository:
-```sh
-git clone https://github.com/fingerprintjs/fingerprint-device-intelligence-android-demo.git
-```
-2. Open up the cloned repository in Android Studio
-3. Open up the `app/src/main/java/com/fingerprintjs/android/fpjs_pro_demo/constants/Credentials.kt` file and assign your Public API Key and Endpoint URL accordingly:
-```kotlin
-import com.fingerprintjs.android.fpjs_pro.Configuration
-
-object Credentials {
-    val apiKey: String = "your_api_key"
-    val endpointUrl: String = Configuration.Region.US.endpointUrl
-}
-```
-You can find the Public API Key in your [dashboard](https://dashboard.fingerprint.com/) at `App Settings` > `API Keys`.
+    ```sh
+    git clone https://github.com/fingerprintjs/fingerprint-device-intelligence-android-demo.git
+    ```
+2. Open the cloned repository in Android Studio.
+3. Open the file `app/src/main/java/com/fingerprintjs/android/fpjs_pro_demo/constants/Credentials.kt` and replace the value for apiKey with your Public API Key. You can find the Public API Key in your [dashboard](https://dashboard.fingerprint.com/) at `App Settings` > `API Keys`.
+   ```kotlin
+    import com.fingerprintjs.android.fpjs_pro.Configuration
+    
+    object Credentials {
+        val apiKey: String = "your_api_key"
+    }
+    ```
+4. Replace the value for endpointUrl with one of our [endpoints](https://dev.fingerprint.com/docs/android-sdk#region-1) or with your custom endpoint, as applicable.
+    ```kotlin
+    import com.fingerprintjs.android.fpjs_pro.Configuration
+    
+    object Credentials {
+        val endpointUrl: String = Configuration.Region.US.endpointUrl
+    }
+    ```
+    
 > [!NOTE]
 > The `debug` and `releaseLocalSign` variants of the app contain a button inside the app bar allowing you to stub the response of the Fingerprint Pro SDK. This might be useful in case you don't want to setup your own API key and region, but still want to check out how the UI of the app looks like on your device.
```

Comment:
Is there a name for this button? How does this button look?

Reply:
@leenasudhakar it's an IconButton, so no, it does not have a name. The icon looks like [this](https://materialui.co/icon/bug-report). It's only purpose is to help testing the UI (success/failure/loading states, animations, transitions, etc) of the app without making an actual request to our servers.